discovery-provider/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # requires authentication to Dockerhub
-FROM audius/contracts:1.0.0 as contracts
+FROM audius/contracts:user-replica-set-manager-12-16-20 as contracts
```

Comment:
just publish 2.0.0, v safe to do so

Reply:
consult w/others before doing this

Reply:
per discussion w @dmanjunath setting to v2.0.0-rc1

Reply:
https://hub.docker.com/layers/audius/contracts/v2.0.0-rc1/images/sha256-e5d74d8fafcce0cdc7ed165dfaa2a9a9a5a5ab8ffd3144f8c72cd2f8da5033f5?context=repo